test/test-clusters/src/main/java/org/elasticsearch/test/cluster/local/FipsEnabledClusterConfigProvider.java

```diff
@@ -33,7 +38,14 @@ public void apply(LocalClusterSpecBuilder<?> builder) {
                 .setting("xpack.security.fips_mode.enabled", "true")
                 .setting("xpack.license.self_generated.type", "trial")
                 .setting("xpack.security.authc.password_hashing.algorithm", "pbkdf2_stretch")
-                .keystorePassword("keystore-password");
+                .keystorePassword("keystore-password")
+                .settings(node -> {
```

Comment:
FWIW, there is a `setting()` method that accepts a predicate specifically for this purpose. This could be simplified to:

```java
.setting("xpack.security.fips_mode.required_providers", () -> "[BCFIPS, BCJSSE]", n -> n.getVersion().onOrAfter("8.13.0"));
```

Reply:
thanks .. i missed that originally but [switched](https://github.com/elastic/elasticsearch/pull/104287/commits/8562f277ddcc539e78cb88abd69f6171059f9db2) over to use that api instead.